### PR TITLE
feat: add 'sleep' command

### PR DESCRIPTION
**Pull request by danitw**

### CODE REVIEW

**Comment by TorchedSammy** [comment]
https://github.com/RosetteOS/Rosebush/pull/13/commits/f762e65c1410801c377d004d4a4c31a5a6748145's type should be chore and https://github.com/RosetteOS/Rosebush/pull/13/commits/47ac59e5fde5d25cd1201362a56d0919b585f173's commit message should be changed to `build: add sleep to makefile`

**Comment by TorchedSammy** [comment]
you uploaded the sleep binary since u didnt build it to the ignored bin directory

File: sleep/sleep.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	args := os.Args[1:]
+
+	if len(args) == 0 {
+		fmt.Println("\033[31msleep\033[0m: missing operand")
+		return
+	}
+
+	arg := args[0]
+
+	last := arg[len(arg)-1:]
```
Comment (TorchedSammy) [inline sleep/sleep.go:20]: space between binary ops (`-`)

File: sleep/sleep.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+	var temp int64
+
+	var err error
+
+	if len(arg) > 1 {
+		temp, err = strconv.ParseInt(arg[:len(arg)-1], 10, 64)
```
Comment (TorchedSammy) [inline sleep/sleep.go:27]: space between binary ops

File: sleep/sleep.go [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	if len(args) == 0 {
+		fmt.Println("\033[31msleep\033[0m: missing operand")
+		return
+	}
+
+	arg := args[0]
+
+	last := arg[len(arg)-1:]
+
+	var temp int64
```
Comment (TorchedSammy) [inline sleep/sleep.go:22]: this var can be renamed to duration

**Review by TorchedSammy — CHANGES_REQUESTED** [review]
mostly style changes
also please remove the extra newline between variable declarations

**Comment by danitw** [comment]
> you uploaded the sleep binary since u didnt build it to the ignored bin directory; delete it

i did not understand

File: sleep/sleep.go (hunk 1 above)
Comment (danitw) [reply sleep/sleep.go:20]: fmt don't let

File: sleep/sleep.go (hunk 2 above)
Comment (danitw) [reply sleep/sleep.go:27]: fmt don't let

**Comment by TorchedSammy** [comment]
this pr is a little broken: makefile has a conflict and commits from `rm` are included

**Pull request closed without merging**